chromium/preloadlist/preloadlist.go

```diff
@@ -47,6 +47,33 @@ type PreloadList struct {
 	Entries []Entry `json:"entries"`
 }
 
+// PolicyType represents the policy under which the domain is a part of the preload list
+type PolicyType string 
+
+// Possible PolicyType values are as defined by 
+// https://source.chromium.org/chromium/chromium/src/+/main:net/http/transport_security_state_static.json
+const (
+	// test domains
```

Comment:
Style: Capitalization should be constant on all these comments, since most read like complete sentences, I'd suggest capitalizing (and adding periods at the end) all of them.

Reply:
Done.

Reply:
"test domains" still shows up uncapitalized for me, and "Domains with an unspecified policy type" is missing a period, maybe you missed a commit to push?

Reply:
Done.